src/encoded/static/components/globals.js

```diff
@@ -44,6 +44,13 @@ documentViews.preview = new Registry();
 documentViews.file = new Registry();
 documentViews.detail = new Registry();
 
+// UNICODE entity codes, needed in JSX string templates. Each property named after the equivalent
+// HTML entity. Add new entries to this object as needed.
+export const uc = {
```

Comment:
would it be better to call this "unicode" or something else more descriptive. 

Where is "uc" used? There are only two references and no involves extracting data from "uc"

Reply:
This needs to be used inline, embedded with a string, and spelling it out could generate complaints. It's used in line 59 of toggle.js.